addons/account/models/account_reconcile_model.py

```diff
@@ -368,6 +368,25 @@ class AccountReconcileModel(models.Model):
     show_decimal_separator = fields.Boolean(compute='_compute_show_decimal_separator')
     number_entries = fields.Integer(string='Number of entries related to this model', compute='_compute_number_entries')
 
+    def _trigger_auto_reconcile(self):
+        # trigger the auto-validate cron on record save if at least one record has auto_reconcile set to True
+        if any(self.mapped('auto_reconcile')):
+            auto_reconcile_cron = self.env.ref('account_accountant.auto_reconcile_bank_statement_line', raise_if_not_found=False)
+            if auto_reconcile_cron:
+                auto_reconcile_cron._trigger()
+
+    # ORM overrides
+    @api.model_create_multi
+    def create(self, vals_list):
+        records = super().create(vals_list)
+        self._trigger_auto_reconcile()
+        return records
+
+    def write(self, values):
+        result = super().write(values)
+        self._trigger_auto_reconcile()
+        return result
```

Comment:
outdated, see `run_auto_reconciliation`

Reply:
I moved it to enterprise and used  `run_auto_reconciliation` can you check